cmd/stack_package_test.go

```diff
@@ -220,6 +221,262 @@ func TestTemplatingFilePermissions(t *testing.T) {
 
 }
 
+func TestPackageNoStackYaml(t *testing.T) {
+
+	// rename stack.yaml to test
+	stackPath := filepath.Join("..", "cmd", "testdata", "starter")
+	stackYaml := filepath.Join(stackPath, "stack.yaml")
+	newStackYaml := filepath.Join(stackPath, "test")
+
+	err := os.Rename(stackYaml, newStackYaml)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer func() {
+		err = os.Rename(newStackYaml, stackYaml)
+		if err != nil {
+			t.Fatal(err)
+		}
+	}()
+
+	// run stack package
+	args := []string{"stack", "package"}
+	_, err = cmdtest.RunAppsodyCmd(args, stackPath, t)
+
+	if err == nil { // stack package will fail as stack.yaml file does not exist
+		t.Fatal(err)
```

Comment:
Could improve error messages here as this is essentially just printing a nil value.